### DISCUSSION TITLE
[Feature] Photo grid asset grouping on web

### DISCUSSION THREAD

**Discussion by bramvdzee** [immich-app/immich#7618] [Feature Request]
### The feature

The mobile app allows for asset grouping per month + day, per day and automatic. I'd be awesome if this could be implemented for the web interface as well.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by advert665** [comment]
A photo grid setting like on the mobile app would be great, I'd really like a month view for the photo grid rather than the day view it has now. Also a 'zoom' slider for the number of photos per row would be great.

**Comment by bishrl** [comment]
I also think this would be a very useful feature. It was the first thing I wanted to do after installing. That said, what a great system for allowing my family to get rid of freaking iCloud storage costs. Thanks for all you do.

**Comment by TarcisioMenezes** [comment]
I’d like to see that implemented too!! :)

**Comment by Didier3L** [comment]
I think these display options would be useful for the web as you can do with the mobile version
